BREEZE.rst

```diff
@@ -492,6 +492,8 @@ Now with the remaining part, replace every ``dash("-")`` with a ``dot(".")``.
 Example:
 If the provider name is ``apache-airflow-providers-cncf-kubernetes``, it will be ``cncf.kubernetes``.
 
+Note: For building docs for apache-airflow-providers index, use dot``(.)`` as short hand operator.
```

Comment:
Maybe `providers` is a better choice?

Reply:
Or `providers-index`  ? (Since we have `all-providers` already.

Reply:
I don't think the dot is the best choice. `providers-index` looks better.

Reply:
Sure sure. Shall I make it `providers.index`?
This is because we are following the dot syntax now instead of dashes in short hand names

Reply:
Sure sure, makes sense